.github/workflows/ci.yml

```diff
@@ -666,19 +665,31 @@ jobs:
         run: gcloud alpha storage cp "${BUILD_DIR_GCS_ARTIFACT}" "${BUILD_DIR_ARCHIVE}"
       - name: "Extracting build dir archive"
         run: tar -xf "${BUILD_DIR_ARCHIVE}"
```

Comment:
We only need the host tools here. We should be able to extract only the install directory, see https://github.com/iree-org/iree/blob/1feec32406e54f46501b36c84f2df11d35a602b3/.github/workflows/ci.yml#L595

Reply:
Addressed in https://github.com/iree-org/iree/pull/10651